src/components/DetailsPage.jsx

```diff
@@ -20,7 +20,7 @@ function DetailsPage() {
           (countrie) => (
             <div key={nanoid()} className="country-card">
               <div className="home-container">
-                <NavLink to="/"><img src={home} alt="home" className="home" /></NavLink>
+                <NavLink to="/countries-database/"><img src={home} alt="home" className="home" /></NavLink>
```

Comment:
- [ ] Hey great job on your project .please kindly make sure all your unit tests are passed when I run the text all your tests failed kindly fix this as the texting is mandatory in the project requirement and it's essential for the app to ensure how the app is functioning properly 

![Screenshot 2023-09-24 032435](https://github.com/CarlosZ96/countries-database/assets/92458236/b0fc3d61-e6c1-415b-b89e-5b4b0dd77349)

- [ ] kindly fix and recheck your project the functionality and unit test need to be fixed as this will give you proper strong confidence in your apps.